### PR TITLE
Expose and prefix JSON type to avoid conflict with other libraries

### PR DESCRIPTION
**Pull request by AlexGaspar**
Implement #343 ideas

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -5,5 +5,6 @@ module.exports = {
   typeMapper: require('./typeMapper'),
   attributeFields: require('./attributeFields'),
   simplifyAST: require('./simplifyAST'),
-  relay: require('./relay')
+  relay: require('./relay'),
+  GraphQLJSON: require('./types/jsonType')
```
Comment (mickhansen) [inline src/index.js]: just JSONType maybe? :)

Comment (AlexGaspar) [inline src/index.js]: Sure! I've updated the pr :)

**Comment by AlexGaspar** [comment]
Is there something preventing this from being merged?

**Comment by mickhansen** [comment]
@AlexGaspar Nope, not at all, i forgot to merge & release, will do it tomorrow.

**Comment by mickhansen** [comment]
v5.0.0

**Pull request merged**